katas/metal-slug/mateo/weapon.js

```diff
@@ -0,0 +1,6 @@
+export class Weapon {
+    constructor(atackDamage, range) {
+        this.atackDamage = atackDamage;
+        this.range = range;
```

Comment:
I think `range` was not part of the initial requirements, in such case, it is better if you try to avoid adding code that is not directly related to requirements given, it can give you unnecessary distractions